en/docs/learn/using-the-end-user-dashboard.md

```diff
@@ -263,3 +263,31 @@ New users can do the following actions from this screen:
     attributes and the corresponding service will not be able to receive
     those attributes in new authentication requests.  
     ![consent-expiry](../assets/img/using-wso2-identity-server/consent-expiry.png) 
+
+### Terminating user sessions
+
+You can click on **View details** in **My Login Sessions** to view your active sessions. 
```

Comment:
To view the active sessions, under the **My Login Sessions**, click **View Details**.